src/NewRecipe.css

```diff
@@ -0,0 +1,65 @@
+.New {
+  margin-bottom: 50px auto;
```

Comment:
can I ask what this specifically does? margin-bottom: 50px auto; is the same as margin-bottom: auto; right?

https://www.w3schools.com/cssref/tryit.asp?filename=trycss_margin-bottom2

I tested it here just to double check. I'm pretty sure this isn't a thing. Happy to be wrong though